### DISCUSSION TITLE
privacy.resistFingerprinting.letterboxing and minMaxCloseButton

### DISCUSSION THREAD

**Discussion by Orriky** [benzBrake/FirefoxCustomize#23] [Ideas]
Hi,

I'd like you port xiaoxiaoflood [minMaxCloseButton.uc.js](https://github.com/xiaoxiaoflood/firefox-scripts/blob/master/chrome/minMaxCloseButton.uc.js) script in alice0775 environment.
Also, I'd like you improve this script by applying the window size and position at Firefox startup, since Firefox doesn't remember them when privacy.resistFingerprinting.letterboxing is set to true.

**Comment by benzBrake** [comment]
You can run this script via  https://github.com/benzBrake/FirefoxCustomize/blob/master/userChromeJS/000-xiaoxiaoflood.uc.js

I try to set privacy.resistFingerprinting.letterboxing to true, My firefox can rmember the position.

**Comment by Orriky** [comment]
Thanks!!!,
I use [arkenfox user.js](https://github.com/arkenfox/user.js) and other custom settings (privacy.resistFingerprinting.letterboxing is set to true) and unfortunately Firefox doesn't remember the window size. This is annoying

**Comment by Orriky** [comment]
I found the setting that prevents Firefox to remember size and position
it is **_privacy.resistFingerprinting_** set to **true**
